client/package.json

```diff
@@ -67,6 +67,7 @@
     "@types/react-dom": "^18.2.17",
     "@typescript-eslint/eslint-plugin": "^6.13.2",
     "@typescript-eslint/parser": "^6.13.2",
+    "audiosprite": "^0.7.2",
```

Comment:
audiosprite isn't used in this package

Reply:
Are you sure ? The first part of the command line is to moving in the client folder

Reply:
nvm